backend/api/communication/src/main/java/co/airy/core/api/communication/lucene/DocumentMapper.java

```diff
@@ -30,7 +33,13 @@ public Document fromConversationIndex(ConversationIndex conversation) {
         document.add(new StoredField("unread_message_count", conversation.getUnreadMessageCount()));
 
         for (Map.Entry<String, String> entry : conversation.getMetadata().entrySet()) {
-            document.add(new TextField("metadata." + entry.getKey(), entry.getValue(), Field.Store.YES));
+            if (!entry.getKey().equals(MetadataKeys.TAGS)) {
+                document.add(new TextField("metadata." + entry.getKey(), entry.getValue(), Field.Store.YES));
+            }
+        }
```

Comment:
I did this for completeness but to be honest I would remove the indexing all together for the metadata and rework this part when we know what kind of queries we'd need to support

Reply:
Agreed

Reply:
👍